pkg/crc/machine/hyperkit/driver_darwin.go

```diff
@@ -34,7 +33,7 @@ func CreateHost(config config.MachineConfig) *hyperkitDriver {
 		},
 		Memory:      config.Memory,
 		CPU:         config.CPUs,
-		UUID:        uuid.NewUUID().String(),
+		UUID:        "c3d68012-0208-11ea-9fd7-f2189899ab08",
```

Comment:
why this specific UUID? I guess it's a random one? I'd mention it in the commit log.

Reply:
@cfergeau This is just fix one, we need to chose one so I used this one :) 

Sure I will put that in the commit logs.